### PR TITLE
enable SMTPS: SMTP over direct TLS connection

### PR DESCRIPTION
**Pull request by zimmersi**
- enable SMTPS for ActionMailer, to be able to send mails over direct TLS connections, usually port 465

### CODE REVIEW

**Comment by zimmersi** [comment]
seems that there are several issues connected, that get closed but not solved!

#2363 
#1881 
#1195

File: Gemfile [hunk 1]
```diff
@@ -86,6 +86,7 @@ gem 'pluck_to_hash', '~> 1.0.2'
 gem 'local_time', '~> 2.1.0'
 
 group :production do
+  gem 'sqlite3', '~> 1.3.6'
```
Comment (mabras) [inline Gemfile:89]: You should remove this.

Comment (zimmersi) [reply Gemfile]: ok, done

**Comment by farhatahmad** [comment]
Can you rebase to master and then run `bundle exec rubocop -A` before pushing?

**Comment by zimmersi** [comment]
hi @farhatahmad,

I get
`invalid option: -A`

do you mean
`-a, --auto-correct               Auto-correct offenses.`
?

**Review by farhatahmad — APPROVED** [review]

**Pull request merged**